src/server/DeviceServer.js

```diff
@@ -837,6 +850,28 @@ class DeviceServer {
       this._eventPublisher.publish(eventData);
     });
   }
+  publishIPCEvent = (
```

Comment:
Yes, i can do this, but you need to remember this attribute if you need it. Based on the methodname you know what happens. That was the idea 

But I will look again tomorrow